java/opendcs/src/main/java/decodes/tsdb/algo/ScalerAdder.java

```diff
@@ -10,96 +10,71 @@
 import decodes.tsdb.MissingAction;
 import decodes.tsdb.ParmRef;
 import decodes.tsdb.VarFlags;
-
-//AW:IMPORTS
-//AW:IMPORTS_END
-
-//AW:JAVADOC
-/**
-Takes up to 10 input values labeled input1 ... input10. Multiplies
-them by coefficients supplied in properties coeff1 ... coeff10.
-Adds them together and produces a single output labeled 'output'.
-Values not assigned by computation are ignored.
-All coefficients default to 1.0 if not supplied.
- */
-//AW:JAVADOC_END
+import org.opendcs.annotations.PropertySpec;
+import org.opendcs.annotations.algorithm.Algorithm;
+import org.opendcs.annotations.algorithm.Input;
+import org.opendcs.annotations.algorithm.Output;
+
+@Algorithm(description ="Takes up to 10 input values labeled input1 ... input10. Multiplies\n" +
+"them by coefficients supplied in properties coeff1 ... coeff10.\n" +
+"Adds them together and produces a single output labeled 'output'.\n" +
+"Values not assigned by computation are ignored.\n" +
+"All coefficients default to 1.0 if not supplied." )
 public class ScalerAdder extends AW_AlgorithmBase
 {
-//AW:INPUTS
-	public double input1;	//AW:TYPECODE=i
-	public double input2;	//AW:TYPECODE=i
-	public double input3;	//AW:TYPECODE=i
-	public double input4;	//AW:TYPECODE=i
-	public double input5;	//AW:TYPECODE=i
-	public double input6;	//AW:TYPECODE=i
-	public double input7;	//AW:TYPECODE=i
-	public double input8;	//AW:TYPECODE=i
-	public double input9;	//AW:TYPECODE=i
-	public double input10;//AW:TYPECODE=i
-	String _inputNames[] = { "input1", "input2", "input3", "input4", "input5", 
+	@Input public double input1;	//AW:TYPECODE=i
```

Comment:
Okay, so on this one, ignore the comment about keeping `@Input` on the line above. While technically not the standard, in this case due to volume it's more readable.